### DISCUSSION TITLE
Create method is not creating new document

### DISCUSSION THREAD

**Discussion by SahasSaurav** [Automattic/mongoose#10218] [Q&A] [answered]
Create method is not creating the new document but able to find the document and even able to modify pre existing document
the code link[https://github.com/SahasSaurav/mern_erp/blob/main/backend/controller/userController.js]
my repo link is https://github.com/SahasSaurav/mern_erp
and current mongoose version is 5.12.2

**Comment by vkarpov15** [comment, marked as answer]
What do you mean by "Create method is not creating the new document" - is there some sort of error message?

**Reply by SahasSaurav** [reply]
I got was able to slove the bug  actually I was using multiple pre hook on save  instead using one pre hook on save with different functions
